tpdatasrc/tpgamefiles/scr/spell_utils.py

```diff
@@ -926,11 +936,14 @@ class AuraSpellHandleModifier(SpellDismissConcentrationFunctions):
     #AuraSpellHandleModifier have at least 6 arguments:
     #spellId, duration, bonusValue, spellEventId, spellDc, empty
     #
+    #particlesId are expected as arg6 for the target particles
```

Comment:
IIRC there's a standardized pfx field in the spell fields... did you try using that?